<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonFormClass</class>
 <widget class="QMainWindow" name="PersonFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PersonForm</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" rowspan="3" colspan="2">
     <widget class="QListWidget" name="listWidgetEvents"/>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Add event</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QLineEdit" name="lineEditName"/>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLineEdit" name="lineEditDesc"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Latitude:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLineEdit" name="lineEditLat"/>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Longitude:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLineEdit" name="lineEditLong"/>
       </item>
       <item row="4" column="0" colspan="4">
        <widget class="QDateEdit" name="dateEditAdd"/>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QListWidget" name="listWidgetParticipants"/>
    </item>
    <item row="1" column="3" rowspan="3">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="2" column="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Update Event</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditDescUpdate"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QDateEdit" name="dateEditUpdate"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonUpdate">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>See near events</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="labelPositon">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="pushButtonGoing">
      <property name="text">
       <string>Going</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PersonForm.qrc"/>
 </resources>
 <connections/>
</ui>
